pkg/plugin/processor/standalone/standalone_test.go

```diff
@@ -119,25 +122,36 @@ func ChaosProcessorSpecifications() sdk.Specification {
 			config.ValidationInclusion{List: []string{"success", "error", "panic"}},
 		},
 	}
+
+	dummyProcessor := sdk.NewProcessorFunc(sdk.Specification{}, nil)
+	spec, err := sdk.ProcessorWithMiddleware(dummyProcessor, sdk.DefaultProcessorMiddleware()...).Specification()
```

Comment:
Here we create a dummy processor just to fetch the specification, which will contain the additional middleware parameters.